docs(gen): tutorials requeriments update part ii (#2924)
scaleway/docs-content: tutorials/first-steps-linux-command-line/index.mdx

<Lightbox src="scaleway-terminal.webp" alt="" />

When you are faced with operating a machine via the command line for the first time, this can be confusing and frustrating. The aim of this document is to is to show you some of the basic principles of using a command line instead of a graphical interface. We will guide you through a series of hands-on exercises to practice some of the main commands you need to get started.

> In this document, we cover commands for a **Linux** command line. The commands for a Windows command line are different, so if you are using Windows you should consult [their own documentation](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands). In the context of virtual machines and servers, a Linux-based OS is generally the go-to choice. Unlike Windows, Linux OSes do not require paid-for licenses, they are free and open source.

Reviewer: ```suggestion
In this document, we cover commands for a **Linux** command line. If you are using a Windows command line, we recommend that you [refer to the Windows documentation](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands), as the commands differ. In the context of virtual machines and servers, a Linux-based OS is generally the go-to choice. Unlike Windows, Linux OSes do not require paid-for licenses, they are free and open source.
```

When you first install Linux, the only user that exists is **root**, which has inherent "superuser" administrative powers. It is not generally advisable to stay permanently logged in as the superuser, for reasons of security. Instead, you should [create a "normal" user account](#creating-and-logging-into-new-user-accounts-adduser-usermod-and-su) and use that when carrying out operations on your machine.

But as a "normal" user, you will not have permissions for everything, such as updating and upgrading software. In order to carry out this kind of operation, you will need to prefix your command with `sudo` (**su**peruser **do**).

> You are usually prompted to enter a password when you enter a `sudo` command, to ensure that you have the right permissions. Once you have entered your password once, you will not be asked to enter it on subsequent `sudo` commands for a little while.

Reviewer: ```suggestion
You are usually prompted to enter a password when you enter a `sudo` command, to ensure that you have the right permissions. After you have entered your password once, you will not be asked to enter it on subsequent `sudo` commands for a little while.
```

Carry on to the next section to see how to update and upgrade your system's software, using `sudo`.

## Updating the system and installing new applications - apt

You can use `apt`, the command for the **A**dvanced **P**ackage **T**ool, to install, update, delete, and manage software packages on Linux systems like Ubuntu and Linux Mint.

> By 'package' we are referring essentially to **software**. On Linux, software is usually built as a package, distributed across remote repositories, and managed by users like you via package managers on their local machine.

Reviewer: ```suggestion
In this context, "package" essentially refers to **software**. On Linux, software is usually built as a package, distributed across remote repositories, and managed by users like you via package managers on their local machine.
```
